proxies/sandbox/apiproxy/proxies/default.xml

```diff
@@ -1,41 +1,65 @@
 <ProxyEndpoint name="default">
     <Flows>
-      <Flow name="OptionsPreFlight">
+        <Flow name="OptionsPreFlight">
             <Request/>
             <Response>
                 <Step>
                     <Name>AssignMessage.AddCors</Name>
                 </Step>
             </Response>
-        <Condition>request.verb == "OPTIONS" AND request.header.origin != null AND request.header.Access-Control-Request-Method != null</Condition>
-      </Flow>
-      <Flow name="AddPayloadToPing">
-        <Description/>
-        <Request/>
-        <Response>
-          <Step>
-            <Name>AssignMessage.AddPayloadToPing</Name>
-          </Step>
-        </Response>
-        <Condition>(proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
-      </Flow>
-      <Flow name="Flow.AuthApiKey">
-        <Request>
-          <Step>
-            <Name>VerifyApiKey.XApiKey</Name>
-          </Step>
-        </Request>
-        <Response/>
-        <Condition>(proxy.pathsuffix MatchesPath "/_status") and (request.verb = "GET")</Condition>
-      </Flow>
+            <Condition>request.verb == "OPTIONS" AND request.header.origin != null AND request.header.Access-Control-Request-Method != null</Condition>
+        </Flow>
+        <Flow name="AddPayloadToPing">
+            <Description/>
+            <Request/>
+            <Response>
+                <Step>
+                    <Name>AssignMessage.AddPayloadToPing</Name>
+                </Step>
+            </Response>
+            <Condition>(proxy.pathsuffix MatchesPath "/_ping") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
+        </Flow>
+        <Flow name="StatusEndpoint">
+            <Description/>
+            <Request>
+                <Step>
+                    <Name>KeyValueMapOperations.GetSharedSecureVariables</Name>
+                </Step>
+                <Step>
+                    <Condition>(private.apigee.status-endpoint-api-key NotEquals request.header.apikey) or (private.apigee.status-endpoint-api-key Is null)</Condition>
+                    <Name>RaiseFault.401Unauthorized</Name>
+                </Step>
+                <Step>
+                    <Name>ServiceCallout.CallHealthcheckEndpoint</Name>
+                </Step>
+            </Request>
+            <Response>
+                <Step>
+                    <Name>javascript.SetStatusResponse</Name>
+                </Step>
+            </Response>
+            <Condition>(proxy.pathsuffix MatchesPath "/_status") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
+        </Flow>
+        <Flow name="Flow.AuthApiKey">
+            <Request>
+                <Step>
+                    <Name>VerifyApiKey.XApiKey</Name>
+                </Step>
+            </Request>
+            <Response/>
+            <Condition>(proxy.pathsuffix MatchesPath "/_status") and (request.verb = "GET")</Condition>
```

Comment:
I don't think this flow can ever be executed with the new status endpoint flow above. I don't think we use this flow currently so this could probably be removed.